examples/clustermanager/0mq/ZMQCM.jl

```diff
@@ -1,6 +1,10 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
-using ZMQ
+# the 0mq clustermanager depends on package ZMQ. For testing purposes, at least
+# make sure the code loads without it.
+try
+    using ZMQ
+end
```

Comment:
revert?

Reply:
This test is intended to work without ZMQ installed. Previously this was handled by putting a fake empty `ZMQ` module in Main.

Reply:
Could we now always just put a fake module in `Base.__toplevel__` instead?

Reply:
I wouldn't encourage that. We would need a better official way to mock out packages. Would probably need more design to fully work with pkg3.